rfc/ipni-w3c.md

```diff
@@ -0,0 +1,346 @@
+# W3 IPNI Indexing RFC
+
+## Authors
+
+- [gammazero], [olizilla]
+
+## Goals
+
+A user can use a CID that identifies data stored on w3c to lookup the location where the data can be retrieved and content claims describing that data.
+
+The data's location can change and content claims about the data can change, at anytime during the data's storage lifetime, and queried for the data CIDs will be able to get the most locations and claims.
+
+This RFC describes how IPNI indexing can be used to lookup data locations and content claims for any CID within a CAR stored in w3c.
+
+## Abstract
+
+A user stores data as a CAR file. This CAR contains a set of multihashes for individual pieces of data within the CAR. 
```

Comment:
While that is the case today I would highly encourage to reframe this differently. That is users store content as opaque blobs containing series of blocks addressed vie multihashes. It is still true but allows us to treat CARs as currently implementation detail